vault/resource_aws_secret_backend_static_role_test.go

```diff
@@ -0,0 +1,55 @@
+package vault
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+	"github.com/hashicorp/terraform-provider-vault/testutil"
+	"testing"
+)
+
+func TestAccAWSSecretBackendStaticRole(t *testing.T) {
+	mount := acctest.RandomWithPrefix("tf-aws-static")
+	a, s := testutil.GetTestAWSCreds(t)
+	username := testutil.SkipTestEnvUnset(t, "AWS_STATIC_USER")[0]
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testutil.TestAccPreCheck(t)
+			SkipIfAPIVersionLT(t, testProvider.Meta(), provider.VaultVersion114)
+		},
+		ProviderFactories: providerFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAWSStaticReourceConfig(mount, a, s, username),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("vault_aws_secret_backend_static_role.role", "name", "test"),
```

Comment:
suggest moving `"vault_aws_secret_backend_static_role.role"` out to a constant like `resourceName`, and updating the field names to use constants 😄 
```suggestion
					resource.TestCheckResourceAttr("vault_aws_secret_backend_static_role.role", consts.FieldName, "test"),
```